src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -123,8 +123,8 @@ public void Crossgen(string outputPath)
             {
                 foreach (var asset in package.Assets)
                 {
-                    var subDir = CreateSubDirectory(outputPath, package, asset);
-                    var targetPath = Path.Combine(subDir.FullName, asset.FileName);
```

Comment:
`package.Assets` -- are these the DLLs and such in the package? If so, this is incorrect. Assets have original case. The things have have lowercase are things that have the package version and ID baked into the file name.